Files/Views/MainPage.xaml.cs

```diff
@@ -98,18 +93,20 @@ public void MultitaskingControl_CurrentInstanceChanged(object sender, CurrentIns
             SidebarAdaptiveViewModel.PaneHolder = e.CurrentInstance as IPaneHolder;
             SidebarAdaptiveViewModel.PaneHolder.ActivePaneChanged += PaneHolder_ActivePaneChanged;
             SidebarAdaptiveViewModel.NotifyInstanceRelatedPropertiesChanged((e.CurrentInstance.TabItemArguments?.NavigationArg as PaneNavigationArguments).LeftPaneNavPathParam);
-            UpdateStatusBarProperties();
+            UpdateSidebarProperties();
+            UpdateNavToolbarProperties();
             e.CurrentInstance.ContentChanged -= TabItemContent_ContentChanged;
             e.CurrentInstance.ContentChanged += TabItemContent_ContentChanged;
         }
 
         private void PaneHolder_ActivePaneChanged(object sender, EventArgs e)
         {
             SidebarAdaptiveViewModel.NotifyInstanceRelatedPropertiesChanged(SidebarAdaptiveViewModel.PaneHolder.ActivePane.TabItemArguments.NavigationArg.ToString());
-            UpdateStatusBarProperties();
+            UpdateSidebarProperties();
+            UpdateNavToolbarProperties();
         }
 
-        private void UpdateStatusBarProperties()
+        private void UpdateSidebarProperties()
```

Comment:
Was there rationale behind this name change?

Reply:
Thanks for catching that, I must've mixed up status bar with sidebar xD